docs/base-app/agents/getting-started.mdx

```diff
@@ -7,186 +7,100 @@ sidebarTitle: 'Getting Started'
 
 This guide will walk you through creating, testing, and deploying your first XMTP messaging agent. By the end, you'll have a fully functional agent that can send and receive messages on the XMTP messaging network.
```

Comment:
Should this doc just point to https://docs.xmtp.org/agents/get-started/build-an-agent instead of repeating the info?

I think the only Base-specific info is getting a base name?